cmd/obslytics/export.go

```diff
@@ -21,55 +26,42 @@ import (
 )
 
 func registerExport(m map[string]setupFunc, app *kingpin.Application) {
-	cmd := app.Command("export", "Export Observability Data into popular analytics formats.")
+	cmd := app.Command("export", "Export observability data into popular analytics formats.")
 	inputFlag := extflag.RegisterPathOrContent(cmd, "input-config", "YAML for input configuration.", true)
 	// TODO(inecas): add support for output configuration.
 	outputFlag := extflag.RegisterPathOrContent(cmd, "output-config", "YAML for output configuration.", false)
 
-	seriesParams := input.SeriesParams{}
-	cmd.Flag("match", "Name of the metric to export").Required().StringVar(&seriesParams.Matcher)
+	// TODO(bwplotka): Describe more how the format looks like.
+	matchersStr := cmd.Flag("matchers", "Metric matcher for metrics to export (e.g up{a=\"1\"}").Required().String()
```

Comment:
What's the use-case for multiple matchers?

Reply:
Essentially OR to have multiple metrics in parquet file. Also I don't get why we treat __name__ label (metric name) specially in parquet. Don't we have to add that to file as well?

Reply:
I'm not sure I can think of a good use-case for `OR` on multiple metrics. The single parquet file represents a single table with specific schema: for multiple metrics, I would rather go for two separate exports.

> Don't we have to add that to file as well?
I don't thing I understood question properly, could you rephrase?

Reply:
Orthogonal to those questions, `--match` would be more consistent with other tools in the ecosystem (https://manpages.debian.org/testing/prometheus/promtool.1.en.html)

Reply:
multiple matchers to form a single query makes sense to me, but I don't get how you can do multiple metrics in a single query. You could do `{__name__="a|b"}` but that doesn't make sense in terms of other labels. 
I think we should make multiple corresponding queries for multiple metrics, this might also be a good way to keep query sizes small.

Reply:
>> Don't we have to add that to file as well?

> I don't thing I understood question properly, could you rephrase?

To me if I read this correctly: https://parquet.apache.org/documentation/latest/#file-format, this file expects columns. Metric name is just another column `__name__`. Given also that this recommendation: https://www.dremio.com/tuning-parquet/#:~:text=The%20official%20Parquet%20documentation%20recommends,SET%20command%20on%20the%20store. I expect those file to be up to 1GB, which means even thousands of metrics in single parquet file.

Also if we skip metric name in parquet file how do we even know what metric name this parquet file corresponds to?

I feel it's expected from this to actually add the label __name__. It's then up to obslytics user to choose if they want one or more metrics per file (: WDYT?

>  Orthogonal to those questions, --match would be more consistent with other tools in the ecosystem (https://manpages.debian.org/testing/prometheus/promtool.1.en.html)

Agree, fair point, will revert.

Reply:
> I think we should make multiple corresponding queries for multiple metrics, this might also be a good way to keep query sizes small.

Again, it's up to user what it selects. Thanos/Prometheus can handle requests spanning across hundreds of thousands of series

Reply:
Also this adding metric name as separate column etc - let's aim for those (if we agree on this) in separate PR, this is already big (:

Reply:
I understand your question now: yes, if we allow multiple metrics in a single parquet file, the name would be required. But at least from how I see parquet files being used, it really don't see a case where I would want one parquet for multiple metrics, unless very very similar labels and diffing mainly in the metric name. Definitely not the main usages I would expect.

I'm ok though on providing the functionality and see how people are using it.

Reply:
Cool, we can always iterate (: